pod/perldsc.pod

```diff
@@ -652,6 +652,10 @@ X<hash of hashes> X<HoH>
      $HoH{flintstones}{$what} = $new_folks{$what};
  }
 
+ # same, but using a slice
+
+ @{ ${HoH}{flintstones} }{ "wife", "pet" } = ( "wilma", "dino" );
```

Comment:
I'd prefer calling it a "hash slice" in the comment (as in `same, but using a hash slice`).

For consistency with the other examples, the empty line between the comment and the code should be removed.

Similarly, the `{ }` around `HoH` shouldn't be there (`@{ $HoH{flintstones} }`).

If you want to completely mimic the previous example, including the use of the `%new_folks` variable, it should look like:
```perl
@{ $HoH{flintstones} }{ keys %new_folks } = values %new_folks;
```